app/static/css/dashboard.css

```diff
@@ -1,225 +1,182 @@
-  /* CSS animations */
-  -webkit-animation: none !important;
-  -moz-animation: none !important;
-  -o-animation: none !important;
-  animation: none !important;
-}
-
 #sidebar.inactive {
-    /*margin-left: -160px;*/
-    /*display: none;*/
     margin-left: -160px;
-
 }
 
 #sidebar .nav-link {
-  font-weight: 500;
-  color: #333;
+    font-weight: 500;
+    color      : #333;
 }
 
 #sidebar i.fa {
-  font-size: .75rem;
-  margin-right: 3px;
-  color: #999;
+    font-size   : .75rem;
+    margin-right: 3px;
+    color       : #999;
 }
 
 #sidebar .nav-link.active {
-  color: #000;
+    color      : #000;
+    font-weight: 800;
 }
 
 #sidebar .sub-bar {
-  font-size: 0.75em;
+    font-size: 0.75em;
 }
 
 #sidebar .nav-link:hover i.fa,
 #sidebar .nav-link.active i.fa {
-  color: inherit;
+    color: inherit;
 }
 
 .sidebar-heading {
-  font-size: .75rem;
-  text-transform: uppercase;
+    font-size     : .75rem;
+    text-transform: uppercase;
 }
 
-
 #content {
-  flex: 999 1 0;
-  min-width: 50%; /* wrap if we're too small*/
-  padding-left: 5px;
-  padding-right: 5px;
-}
-
-
-/* for small screens, flip this meaning, so it's invisible */
-
-@media (max-width: 768px) {
-  .wrapper {
-    display: block;
-  }
-  #sidebar {
-      min-width: auto;
-      max-width: none;
-      min-height: 0px;
-      /*display: none;*/
-  }
-  #sidebar.inactive {
-    margin-left: -1000px;
-    height:0px;
-  }
-
-  .navbar span.pagename { display:none; }
+    flex         : 999 1 0;
+    min-width    : 50%;
+    padding-left : 5px;
+    padding-right: 5px;
 }
 
-
-#sidebar .nav-link {
-
-  font-weight: 500;
-  color: #333;
-}
-
-#sidebar i.fa {
-  font-size: .75rem;
-  margin-right: 3px;
-  color: #999;
-}
-
-#sidebar .nav-link.active {
-  font-weight: 800;
-}
-
-#sidebar .sub-bar {
-  font-size: 0.75em;
+.navbar-brand {
+    padding-top     : .75rem;
+    padding-bottom  : .75rem;
+    font-size       : 1rem;
+    background-color: rgba(0, 0, 0, .25);
+    box-shadow      : inset -1px 0 0 rgba(0, 0, 0, .25);
 }
 
-#sidebar .nav-link:hover i.fa,
-#sidebar .nav-link.active i.fa {
-  color: inherit;
+.navbar .form-control {
+    padding      : .75rem 1rem;
+    border-width : 0;
+    border-radius: 0;
 }
 
-.sidebar-heading {
-  font-size: .75rem;
-  text-transform: uppercase;
+.form-control-dark {
+    color           : #fff;
+    background-color: rgba(255, 255, 255, .1);
+    border-color    : rgba(255, 255, 255, .1);
 }
 
-/*
- * Content
- */
-#content {
-    /* padding-top: 48px;  Space for fixed navbar */
+.form-control-dark:focus {
+    border-color: transparent;
+    box-shadow  : 0 0 0 3px rgba(255, 255, 255, .25);
 }
 
-/*
- * Navbar
- */
-
-.navbar-brand {
-  padding-top: .75rem;
-  padding-bottom: .75rem;
-  font-size: 1rem;
-  background-color: rgba(0, 0, 0, .25);
-  box-shadow: inset -1px 0 0 rgba(0, 0, 0, .25);
+body.deployment-development #navbar-brand {
+    background-color: #fff3cd !important;
+    color           : black;
```

Comment:
Same with these ones - they don't actually make any difference since there are no other more specific rules for these style elements that would be overridden.